modules/core/src/main/scala/iota/internal/toolbelts.scala

```diff
@@ -193,7 +193,7 @@ private[internal] sealed trait TypeListParsers { self: Toolbelt with TypeListAST
   }
 
   private[this] def literalInt(tpe: Type): Either[Id[String], Int] =
-    tpe match {
+    tpe.dealias match {
```

Comment:
This fixes a small bug where type aliases weren't allowed for singleton literals. The fix is needed for tests to pass in this PR.